### ISSUE TITLE
Namespace the channels

### ISSUE THREAD

**Issue by haadcode** [haadcode/orbit#145]
At the moment the channels in orbit are plaintext pubsub topic. We should namespace the channels, eg /orbit.chat/<channel>.

@jbenet's proposal:
"namespace the pubsub channels you use under some long hash (<orbit-hash>/devcon2) or a domain name (<orbit-domain>/devcon2)"

**Comment by theobat** [comment]
What about namespacing channels based on organizations (following the slack-like email links) ? 
i.e. if you have a whatever@ipfs.com you could get access to all the /@ipfs/something channels ... 

I do believe it would be interesting to keep the non name-spaced channels as well though it's an interesting feature for people without need for "private" (whatever that means) discussions

**Comment by haadcode** [comment]
@theobat absolutely! This is the idea of namespacing is to be able to limit access. How exactly that happens, no idea :)

Email could be one. Another could be some key-based system. Another could be a "fully managed" solution with blockchain where the organization controls the access.

> I do believe it would be interesting to keep the non name-spaced channels as well though it's an interesting feature for people without need for "private"

Agreed.
